frontend/src/components/Buttons/TypeButton.vue

```diff
@@ -1,94 +1,84 @@
 <template>
   <v-menu offset-y>
-    <template #activator="{ on, attrs }">
+    <template #activator="{ props }">
       <v-btn
-        v-if="!$vuetify.display.smAndDown && items[model]"
+        v-if="!$vuetify.display.smAndDown && (model == 0 || model[0])"
         class="my-2"
         variant="text"
         rounded
         :disabled="disabled"
-        v-bind="attrs"
-        v-on="on">
-        {{ items[model].name }}
+        v-bind="props">
+        {{
+          model == 0
+            ? items[0].title
+            : items.find((x) => x.value == model[0]).title
+        }}
         <v-icon end>
           <i-mdi-menu-down />
         </v-icon>
       </v-btn>
-      <v-btn
-        v-else
-        :disabled="disabled"
-        class="my-2"
-        icon
-        v-bind="attrs"
-        v-on="on">
+      <v-btn v-else :disabled="disabled" class="my-2" icon v-bind="props">
         <v-icon>
           <i-mdi-eye />
         </v-icon>
       </v-btn>
     </template>
-    <v-list dense>
-      <v-list-group
-        v-model="model"
-        @change="$emit('change', items[model].value)">
-        <v-list-item v-for="item in items" :key="item.value">
-          <v-list-item-title>{{ item.name }}</v-list-item-title>
-        </v-list-item>
-      </v-list-group>
-    </v-list>
+    <v-list
+      v-model:selected="model"
+      :items="items"
+      @update:selected="emit('change', model[0])" />
```

Comment:
I think this is wrong. Model is not an array according to what you have defined.

Reply:
Because the items get loaded in the `v-list` using `:items="items"`, the selected item gets put in an array, when passing it to the parent component, we need to take the first object in that array. Not sure why it does that, but without the `[0]` behind it, the parent component does not recognize the selected item. Might have something to do with https://next.vuetifyjs.com/en/api/v-list/#props-select-strategy . But it is not documented by Vuetify at all.